src/components/Message.vue

```diff
@@ -0,0 +1,35 @@
+<template id="message">
+  <transition name="fade" v-if="visible">
+    <div class="message" v-bind:class="{'error': error, 'good': success}" v-html="text"></div>
+  </transition>
```

Comment:
maybe we could just call good success too? I think we need to get rid of the goods!